src/webui/api/torrentscontroller.cpp

```diff
@@ -425,7 +430,7 @@ void TorrentsController::propertiesAction()
     dataDict[KEY_TORRENT_ID] = torrent->id().toString();
     dataDict[KEY_PROP_TIME_ELAPSED] = torrent->activeTime();
     dataDict[KEY_PROP_SEEDING_TIME] = torrent->finishedTime();
-    dataDict[KEY_PROP_ETA] = static_cast<double>(torrent->eta());
```

Comment:
I wonder why there were previously casts to the double type in several places?

Reply:
I'm not sure but now `QJsonValue(qint64 v)` is the same as `QJsonValue(double v)`.